packages/types/src/documents/global/userGroup.ts

```diff
@@ -0,0 +1,15 @@
+import { Document } from "../document"
+import { User } from "./user"
+export interface UserGroup extends Document {
+  name: string
+  icon: string
+  color: string
+  users: User[]
```

Comment:
Yea, we need it in numerous locations in the new groups frontends

Reply:
Gotcha. tbh it's not ideal this way as we will get out of date information e.g. if a user's name is updated, their email changes etc, then we have to make sure that user is updated inside every group. Are those scenarios catered for?

An improvement would be to only store the id's, and in the front end load the users via an api call i.e. `/api/global/groups/:groupId/users`, ideally paginated. This could mean we could page through the list of users in a group rather than returning potentially thousands of nested, out of date users.